configure.py

```diff
@@ -109,7 +106,7 @@
 
 # Apply arguments
 config.build_dir = args.build_dir
-config.build_dtk_path = args.build_dtk
+config.build_dtk_path = args.dtk
```

Comment:
Let's rename this config field too

Reply:
Like f146c738771fdc679af2bea62ac0b1969f7d4a11?

Reply:
Oh, it should be `_path`, I see.